services/connector/commands/send_transaction.go

```diff
@@ -65,10 +73,49 @@ func (c *SendTransactionCommand) Execute(request RPCRequest) (interface{}, error
 		return "", err
 	}
 
+	if !params.Valid() {
+		return "", ErrSendingParamsInvalid
+	}
+
 	if params.From != dApp.SharedAccount {
 		return "", ErrParamsFromAddressIsNotShared
 	}
 
+	if params.Value == nil {
+		params.Value = (*hexutil.Big)(big.NewInt(0))
+	}
+
+	if params.GasPrice == nil || (params.MaxFeePerGas == nil && params.MaxPriorityFeePerGas == nil) {
```

Comment:
@alaibe basically, you hit the gist of this PR with this question.

That was my thought as well, but I believe that our Status Browser Connect plugin should handle that and request the send transaction action from any wallet (that's what Wallet Connect does), but since Status Browser Connect just forwards dapp's calls it can only work with Status app (as a wallet). So what we have to do here is to pre-process the data (add the necessary parameters on top of the base parameters that are coming from the dapp) and send all that to the wallet for signing.